JavaScript/tests/syntax_test_jsx.jsx

```diff
@@ -1,5 +1,86 @@
 // SYNTAX TEST "Packages/JavaScript/JSX.sublime-syntax"
 
+// comment
```

Comment:
Why duplicate the tests in other files?

Reply:
I just copied the whole bunch of comments from typescript to the other ones and adjusted rerequired stuff. May probably not needed in the ...X files at the moment, but it may not hurt to make sure not to regress something in case special treatment in react comments is needed in future.

I even prevented myself by regressing something by doing so (see the most recent force-pushes) ;-)